doc/help.rst

```diff
@@ -2,38 +2,89 @@
 Help
 ####
 
-You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods.
+You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods. This allows you to access documentation easily directly from Spyder not having to break your workflow.
+
+|
+
+.. raw:: html
+
+	<div style="width:100%;height:0;padding-bottom:56%;position:relative;"><iframe src="https://giphy.com/embed/ghTnHFBPVB5P5kqEYz" width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed" allowFullScreen></iframe></div><p> <a href="https://giphy.com/gifs/ghTnHFBPVB5P5kqEYz">via GIPHY</a></p>
+
+|
+
+
+You can also access Spyder's tutorial from here, which will guide you through some basic steps for using its key features.
+
+.. image:: images/help/help-tutorial.png
+   :alt: Spyder Help pane showing the link for the Spyder's tutorial
+
+|
+
+
+============
+Getting help
+============
+
 Help can be retrieved both by static analysis of open files in the :doc:`editor`, or by dynamically inspecting an object in an :doc:`ipythonconsole`.
+You can trigger help manually by pressing the configurable help shortcut (:kbd:`Ctrl-I` by default), or automatically if enabled when typing a left parenthesis (``(``) after a function or method name.
 
-You can trigger help by manually entering the object's name into the :guilabel:`Object` box, pressing the configurable help shortcut (:kbd:`Ctrl-I` by default), or even automatically if desired when on typing a left parenthesis (``(``) after a function or method name.
-Automatic help can be individually enabled for both the :guilabel:`Editor` and the :guilabel:`Console` under :menuselection:`Preferences --> Help --> Automatic Connections`, and turned on and off dynamically via the :guilabel:`lock` icon in the top right corner of the :guilabel:`Help` pane.
+.. image:: images/help/help-command.png
+   :alt: Spyder Editor and Help panes, with the latter displaying documentation for an object selected in the former.
 
+|
+
+You can also manually enter the object's name into the :guilabel:`Object` textbox at the top of the pane, if :guilabel:`Console` is selected as the :guilabel:`Source`.
+
+.. image:: images/help/help-object-box.png
```

Comment:
It really isn't clear how this illustrates its presumed subject, since the cursor isn't even in the box. If you're going to use this, the cursor should actually be in the box and there show be arrows/circles around the source menu and object text field to clearly indicate this.

Reply:
The GIF currently used as the hero would be a prefect replacement for this image and address all of my concerns much better than just statically annotating it.